app/assets/stylesheets/common.scss

```
@@ -22,12 +28,16 @@
       font-weight: bold;
     }
   }
+  .rounded-circle {
+    width: 100%;
+    height:auto;
+    max-width:150px;
+    background-color: #ccc;
```

Ich wollte da eh nochmal Stylefarben nutzen
```suggestion
    background-color: $mediumgray;
```